docs/extension/guides/set-wallet-recovery.md

```diff
@@ -0,0 +1,63 @@
+---
+sidebar_label: '🦊 Set Wallet Recovery'
```

Comment:
I would not use the fox emoji as it implies MetaMask but the feature is not MetaMask related. Any wallet can work.